.devcontainer/Dockerfile

```diff
@@ -9,3 +9,5 @@ RUN install-tool python 3.11.4
 
 # renovate: datasource=npm
 RUN install-tool yarn 1.22.19
+
+USER ubuntu
```

Comment:
@viceice could be even further up?

Reply:
let me see

Reply:
Nope:

![image](https://github.com/renovatebot/renovate/assets/29582865/e2cb785f-c64f-4aa4-aeb3-dc3a8db48a9e)

install-tool must be ran as root.

Reply:
install-tool can be run without root, but install-apt needs root of cause

Reply:
I did some tests here, even install-tool python requires root. But install-tool yarn doesn't.

Do you want me to move it to before install-tool yarn?

Reply:
no, it's ok. 

python needs a `prepare-tool python` to he able to install without root.